api/resources/views/home.blade.php

```diff
@@ -9,12 +9,28 @@
 
                 <div class="card-body">
                     @if (session('status'))
-                        <div class="alert alert-success" role="alert">
-                            {{ session('status') }}
-                        </div>
+                    <div class="alert alert-success" role="alert">
```

Comment:
This can be deleted as it serves no purpose. It was just here because no one touched the home page, since you did, please remove this.

Reply:
Tik sita eilute ar visa ifa?